src/stylesheet-registry.js

```diff
@@ -171,19 +171,21 @@ export default class StyleSheetRegistry {
   }
 
   getIdAndRules(props) {
-    if (props.dynamic) {
-      const styleId = this.computeId(props.styleId, props.dynamic)
+    const { children, dynamic, id } = props
```

Comment:
can you alias `children` to `css` here?

`const { children: css, dynamic, id } = props`